src/hooks/browser-headers.ts

```diff
@@ -1,26 +1,52 @@
-const http2 = require('http2-wrapper');
+import { URL } from 'url';
+import { Options } from 'got-cjs';
+import http2 = require('http2-wrapper');
```

Comment:
Any reason for `import x = require`? Pretty sure we have esModuleInterop

Reply:
That's the only way to access CJS modules AFAIK. `import from` ES modules.

Reply:
With

```
        "allowSyntheticDefaultImports": true,
        "esModuleInterop": true,
```

I'm getting

```
{ default: [class HeaderGenerator] } // import * as HeaderGenerator from 'header-generator';
Uncaught TypeError: HeaderGenerator is not a constructor
```

Reply:
Drop the `* as`, and just `import HeaderGenerator` and it should work

Reply:
It worked, thanks :D